packages/cli/generators/discover/index.js

```diff
@@ -180,6 +191,66 @@ module.exports = class DiscoveryGenerator extends ArtifactGenerator {
     });
   }
 
+  /**
+   * Prompts what naming convention they would like to have for column names.
+   */
+  promptColNamingConvention() {
+    this.namingConvention = {
+      'LoopBack default camelCase convention (Recommended)': false,
+      'keep property names the same as database column names': true,
```

Comment:
Using boolean value for different choices is a bit confusing. We can use `camelCase` and `noCase`. In the future, maybe we should prompt a list of case conversions, such as:

```
> Camel case (exampleColumn) 
> Pascal case (...)
> No conversion
> Lower case
> Upper case
```

Reply:
I use boolean to represent `disableCamelCase`. When it prompts, it only shows the messages as options 
```
    ❯ LoopBack default camelCase convention (Recommend)  // maps to disableCamelCase = false
    keep property names the same as database column names // maps to disableCamelCase = true
```

I agree that ideally we can prompt a list of different conventions. I think boolean is probably enough for now. How about improving the prompt messages to:
```
❯  camelCase ( Recommended)
No conversion ( same as database column names)
```

Reply:
Do you want to give an example for each choice? Such as `first-name (column)` to `firstName (property)`?

I know your intention to directly map the two choices to `false` and `true`. It's just not very intuitive. Personally, I would rather use `disableCamelCase = (answer !== 'camelCase')`.